packages/zoe/src/contractFacet/types.js

```diff
@@ -98,6 +99,35 @@
  * @returns {Promise<Invitation>}
  */
 
+/**
+ * @typedef {Object} InvitationWithSeat
+ * @property {Promise<Invitation>} invitation
+ * @property {ZCFSeat} zcfSeat
+ * @property  {ERef<UserSeat>} userSeat
+ */
+
+/**
+ * @callback MakeInvitationWithSeat
+ *
+ * Make an invitation and a seat at the same time. The seat is created before
+ * the offer so that the contract can allocate to it or assign results early.
+ * Make a credible Zoe invitation for a particular smart contract
+ * indicated by the `instance` in the extent of the invitation. Zoe
```

Comment:
```suggestion
 * indicated by the `instance` in the value of the invitation. Zoe
```
We also call this "invitation details"

Reply:
I used "invitation details."